src/MagnumPlugins/AssimpImporter/AssimpImporter.h

```diff
@@ -0,0 +1,147 @@
+#ifndef Magnum_Trade_AssimpImporter_h
+#define Magnum_Trade_AssimpImporter_h
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017
+              Vladimír Vondruš <mosra@centrum.cz>
+    Copyright © 2017 Jonathan Hale <squareys@googlemail.com>
+
+    Permission is hereby granted, free of charge, to any person obtaining a
+    copy of this software and associated documentation files (the "Software"),
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included
+    in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+*/
+
+/** @file
+ * @brief Class @ref Magnum::Trade::AssimpImporter
+ */
+
+#include <Magnum/Trade/AbstractImporter.h>
+#include <Magnum/Trade/ObjectData3D.h>
+
+#include "MagnumPlugins/AssimpImporter/visibility.h"
+
+#include <assimp/Importer.hpp>
+#include <assimp/scene.h>
+
+#include <unordered_map>
+
+namespace Magnum { namespace Trade {
+
+/**
+@brief Assimp importer
+
+Imports various formats using [Assimp](http://assimp.org).
+
+Supports importing of scene, object, camera, mesh, texture and image data.
+
+It is built if `WITH_ASSIMPIMPORTER` is enabled when building Magnum Plugins.
+To use dynamic plugin, you need to load `AssimpImporter` plugin from
+`MAGNUM_PLUGINS_IMPORTER_DIR`. To use static plugin, you need to request
+`AssimpImporter` component of `MagnumPlugins` package in CMake and link to
+`MagnumPlugins::AssimpImporter`. See @ref building-plugins, @ref cmake-plugins
+and @ref plugins for more information.
+
+### Behaviour and limitations
+
+#### Material import
+
+- Only materials with shading mode `aiShadingMode_Phong` are supported
+- Only the first diffuse/specular/ambient texture is loaded
+
+
```

Comment:
Mentioning the types of `importerState()` return values here would be nice (look in the OGEX importer for an example).